### PR TITLE
Benji/2234 enter submits forms

### LINKED ISSUE #2234: Pressing enter on action modals should submit forms
**Describe the bug**

Pressing enter on action modals should submit the form. Currently, it does not.

**To Reproduce**

- go to wallet page
- open send modal
- fill in empty form fields
- press `Enter`
- form is not submitted

**Expected behavior**

I would expect the form to be submitted.

**Additional context**

I believe this would be more semantic and accessible HTML.

### PR DESCRIPTION
**Pull request by thebkr7**
Closes #2234 

**Description:**

Allows users to navigate through all forms by pressing enter. 

Thank you! 🚀

---

For contributor:

- [x] Added changes entries. Run `yarn changelog` for a guided process.
- [x] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by thebkr7** [comment]
Would love some feedback as to the error happening in the tests. I can only find examples of cases with the misspelling of custom vue directives which I have none.

File: src/ActionModal/components/DelegationModal.vue [hunk 1]
```diff
@@ -186,6 +190,12 @@ export default {
       this.selectedIndex = 0
       this.amount = null
     },
+    passToParent(func) {
+      this.validateChangeStep = func
+    },
+    enterPressed() {
+      this.validateChangeStep()
```
Comment (faboweb) [inline src/ActionModal/components/DelegationModal.vue]: rather then passing functions around, you can reference the actionModal
```suggestion
      this.$refs.actionModal.validateChangeStep()
```
I recommend that you get rid of the `passToParent` to clean up some code.

Comment (thebkr7) [reply src/ActionModal/components/DelegationModal.vue]: Much simpler. I will update

**Comment by thebkr7** [comment]
The e2e test appears to be the same two that were giving us a timeout error before.

**Comment by faboweb** [comment]
> The e2e test appears to be the same two that were giving us a timeout error before.

They sometimes fail. :/ Ramp up the coverage and we see if the e2e tests run successful then.

**Comment by thebkr7** [comment]
I'd have liked to test if ValidateChangeStep was called in ActionModal but couldn't get the $refs right. Any suggestions?

**Comment by faboweb** [comment]
Nice! Works pretty neat.

> I'd have liked to test if ValidateChangeStep was called in ActionModal but couldn't get the $refs right. Any suggestions?

in which scenario?

**Pull request merged**